package.json

```diff
@@ -20,7 +20,7 @@
   ],
   "scripts": {
     "test": "jest",
-    "lint": "eslint src tests",
```

Comment:
Removed because tests are under ./src. The command will fail with a missing dir in the list.